.github/workflows/build.yml

```diff
@@ -55,3 +55,22 @@ jobs:
             :construction: Work in Progress
             
             ---
+      - name: Create source tarball
+        if: startsWith(github.ref, 'refs/tags/')
+        run: git archive --prefix="cryptomator-${{ github.ref }}/" -o "cryptomator-${{ github.ref }}.tar.gz" ${{ github.ref }}
+      - name: Sign source tarball with key 615D449FE6E6A235
+        if: startsWith(github.ref, 'refs/tags/')
+        run: |
+          echo "${GPG_PRIVATE_KEY}" | gpg --batch --quiet --import
+          echo "${GPG_PASSPHRASE}" | gpg --batch --quiet --passphrase-fd 0 --pinentry-mode loopback -u 615D449FE6E6A235 --detach-sign -a "cryptomator-${{ github.ref }}.tar.gz"
+        env:
+          GPG_PRIVATE_KEY: ${{ secrets.RELEASES_GPG_PRIVATE_KEY }}
+          GPG_PASSPHRASE: ${{ secrets.RELEASES_GPG_PASSPHRASE }}
+      - name: Publish source tarball signature on GitHub Releases
+        if: startsWith(github.ref, 'refs/tags/')
+        uses: softprops/action-gh-release@v1
+        with:
+          fail_on_unmatched_files: true
+          token: ${{ secrets.CRYPTOBOT_RELEASE_TOKEN }}
+          files: |
+            cryptomator-${{ github.ref }}.tar.gz.asc
```

Comment:
Can this be combined with the existing release step?

Reply:
Wow, nice, that really reduces overhead :slightly_smiling_face:, fixed in 319e171

Reply:
just the publishing part. afaik you can't combine `run:` with `uses:` when using a third party action.

what I meant was to reorder the steps, so we can reference the `.tar.gz.asc` in the `softprops/action-gh-release@v1` step:

```yml
      steps:
       - name: Sign source tarball with key 615D449FE6E6A235
         if: startsWith(github.ref, 'refs/tags/')
         run: |
           git archive --prefix="cryptomator-${{ github.ref }}/" -o "cryptomator-${{ github.ref }}.tar.gz" ${{ github.ref }}
           echo "${GPG_PRIVATE_KEY}" | gpg --batch --quiet --import
           echo "${GPG_PASSPHRASE}" | gpg --batch --quiet --passphrase-fd 0 --pinentry-mode loopback -u 615D449FE6E6A235 --detach-sign -a "cryptomator-${{ github.ref }}.tar.gz"
         env:
           GPG_PRIVATE_KEY: ${{ secrets.RELEASES_GPG_PRIVATE_KEY }}
           GPG_PASSPHRASE: ${{ secrets.RELEASES_GPG_PASSPHRASE }}
       - name: Draft a release
         if: startsWith(github.ref, 'refs/tags/')
         uses: softprops/action-gh-release@v1
         with:
           draft: true
           discussion_category_name: releases
          token: ${{ secrets.CRYPTOBOT_RELEASE_TOKEN }}
          generate_release_notes: true
          files: |
            cryptomator-${{ github.ref }}.tar.gz.asc
          body: |-
             :construction: Work in Progress
             
             ---
```